### PR TITLE
[CB-163] Form resets when the Search button is clicked

### PR DESCRIPTION
**Pull request by LWanjiru**
# JIRA TICKET NAME:
CB-163 - [Form resets when the Search button is clicked](https://issues.openmrs.org/browse/CB-163)

## SUMMARY:
Currently, when a user (1) does a query about obs, (2) does a query about encounters, (3) views the results of the obs query, (4) clicks Back.
They see a form with the values entered in their previous search, which can be quite confusing.

After bug fix:
After clicking on 'Search', The data should be saved and the query reset. 
Thus, a user should see an empty form upon clicking the ‘Search’ button. This means that even after viewing results, a user should view a reset form on clicking 'Back'.

### CODE REVIEW

File: app/js/components/tabs/tabcomponents/observationComponent.jsx [hunk 1]
```diff
@@ -132,32 +132,32 @@ export default class ObsFilter extends React.Component {
...
   firstFieldNumeric(){
     return(
-            <div className="form-group" >
-                <label className="col-sm-3 control-label">Which observations?</label>
-                <div className="col-sm-6">
-                    <select 
-                        className="form-control"
-                        value={this.state.timeModifier}
-                        defaultValue="ANY"
-                        name="timeModifier"
```
Comment (dkayiwa) [inline app/js/components/tabs/tabcomponents/observationComponent.jsx:142]: Did you intentionally do all these formatting changes?

Comment (LWanjiru) [reply app/js/components/tabs/tabcomponents/observationComponent.jsx:142]: Yes, I did.  When I  made the pull request, HoundCI pointed out some indentation issues in my files. They occurred after a rebase, to get the changes that had been newly added to  the base branch. I had however missed them while committing, so I went back and fixed them, alongside other styling issues that did not follow eslint best practices.

Comment (dkayiwa) [reply app/js/components/tabs/tabcomponents/observationComponent.jsx:142]: In such cases, we do a separate pull request for just indention or formatting changes, and then another one with the real changes required for this ticket. That way, the reviewer finds it easier to tell what actually changed to fix the reported problem and what was simply unrelated formatting. So can you do the two separate pull requests?

Comment (LWanjiru) [reply app/js/components/tabs/tabcomponents/observationComponent.jsx:142]: Okay, let me work on doing that. Thank you for letting me know about separating the formatting changes and the feature changes when making a PR, I did not know that beforehand.

Comment (LWanjiru) [reply app/js/components/tabs/tabcomponents/observationComponent.jsx:142]: I have separated the two and created a PR for the the formatting, and I have another branch rebased from that with only the changes concerning the task. Please review the ticket CB-167 for merging, so I can make the PR for the other ticket CB-163 with those changes in place.  Thank you.

Comment (dkayiwa) [reply app/js/components/tabs/tabcomponents/observationComponent.jsx:142]: @LWanjiru i would like to start with the pull request having the actual changes without formatting.

Comment (LWanjiru) [reply app/js/components/tabs/tabcomponents/observationComponent.jsx:142]: Okay.

**Comment by LWanjiru** [comment]
This pull request was **closed** in order to give way for the separation of concerns, in regard to creating separate PRs; ie. 
1)  [Have one PR exclusively showing the actual task's changes ](https://github.com/openmrs/openmrs-owa-cohortbuilder/pull/150)
2) [Have another PR handling the format and styling changes](https://github.com/openmrs/openmrs-owa-cohortbuilder/pull/149)

**Pull request closed without merging**